src/com/palmergames/bukkit/towny/TownyFormatter.java

```diff
@@ -378,11 +378,12 @@ private static void getRanks(Town town, List<String> ranklist) {
 
 		if (town.isRuined()) {
 			out.add(Translation.of("msg_time_remaining_before_full_removal", TownRuinSettings.getTownRuinsMaxDurationHours() - TownRuinUtil.getTimeSinceRuining(town)));
-			if (TownRuinSettings.getTownRuinsReclaimEnabled()) {
-				if (TownRuinUtil.getTimeSinceRuining(town) < TownRuinSettings.getTownRuinsMinDurationHours())
-					out.add(Translation.of("msg_time_until_reclaim_available", TownRuinSettings.getTownRuinsMinDurationHours() - TownRuinUtil.getTimeSinceRuining(town)));
-				else 
-					out.add(Translation.of("msg_reclaim_available"));
+			if (TownRuinSettings.getTownRuinsReclaimEnabled()
+					&& TownRuinUtil.getTimeSinceRuining(town) < TownRuinSettings.getTownRuinsMinDurationHours()) {
+				out.add(Translation.of("msg_time_until_reclaim_available", TownRuinSettings.getTownRuinsMinDurationHours() - TownRuinUtil.getTimeSinceRuining(town)));
+			}
+			else {
+				out.add(Translation.of("msg_reclaim_available"));

```

Comment:
If I'm not mistaken this will result in the msg_reclaim_available message being show in situations where getTownsRuinsReclaimEnabled is not true...

Reply:
Ah, yeah... This one is human error on my part. Linter never complained about this one. Should be resolved with next commit.